config/egfr.yaml

```diff
@@ -0,0 +1,84 @@
+# The length of the hash used to identify a parameter combination
```

Comment:
I'm not sure how the files from the EGFR update were added to this branch as new files.  You can delete this and the two new input files on this branch.